src/test/java/com/cloudant/tests/AttachmentsTest.java

```diff
@@ -39,26 +40,15 @@
 import com.cloudant.client.api.model.Response;
 import com.cloudant.tests.util.Utils;
 
-public class AttachmentsTest {
+public class AttachmentsTest extends CloudantClientTestBase{
+
 
-	private static final Log log = LogFactory.getLog(AttachmentsTest.class);
-	
-	private static CloudantClient dbClient;
 	private static Database db;
-	private static Properties props ;
-
-	@BeforeClass
-	public static void setUpClass() {
-		props = Utils.getProperties("cloudant.properties",log);
-		dbClient = new CloudantClient(props.getProperty("cloudant.account"),
-				  props.getProperty("cloudant.username"),
-				  props.getProperty("cloudant.password"));
-		db = dbClient.database("lightcouch-db-test", true);
-	}
 
-	@AfterClass
-	public static void tearDownClass() {
-		dbClient.shutdown();
+	@Before
```

Comment:
Shouldn't this be annotated `@BeforeClass`? I don't see we need to do it before each test. I think this applies to most of the other test classes as well as the DB setup now seems to be annotated `@Before` instead of  `@BeforeClass` in all of them.